package.json

```diff
@@ -17,6 +17,9 @@
     "postlink-dist": "cd dist && rm -rf node_modules",
     "unlink-dist": "cd dist && npm unlink && rm package*",
     "watch": "NODE_ENV=development rollup --watch -c",
+    "i18n-scan": "i18next-scanner --config i18next-scanner.config.js src/**/*.{js,jsx}",
+    "i18n-conv-json": "i18next-conv -l en -s ./src/lib/translations/en/translations.json -t ./src/lib/translations/translations.pot",
+    "i18n-conv-po": "i18next-conv -l $npm_config_lang -s ./src/lib/translations/$npm_config_lang/messages.po -t ./src/lib/translations/$npm_config_lang/translations.json",
```

Comment:
Maybe we will need to have `i18n-conv-po-all` function to compile all the `.po` files for all available languages...Maybe the configured languages could be set here as a config variable https://docs.npmjs.com/cli/v7/configuring-npm/package-json#config